common/persistence/historyStore.go

```diff
@@ -456,13 +456,13 @@ func (m *historyV2ManagerImpl) readHistoryBranch(
 			// In that case we don't validate history continuousness for the first page
 			// TODO: in this case, some events returned can be invalid(stale). application layer need to make sure it won't make any problems to XDC
 			if defaultLastEventID == 0 || token.LastEventID != defaultLastEventID {
-				logger.Error("Corrupted non-continuous event batch",
+				logger.Error("Corrupted non-contiguous event batch",
 					tag.FirstEventVersion(firstEvent.GetVersion()), tag.WorkflowFirstEventID(firstEvent.GetEventId()),
 					tag.LastEventVersion(lastEvent.GetVersion()), tag.WorkflowNextEventID(lastEvent.GetEventId()),
 					tag.TokenLastEventVersion(token.LastEventVersion), tag.TokenLastEventID(token.LastEventID),
 					tag.Counter(eventCount))
 				return historyEvents, historyEventBatches, nil, dataSize, lastFirstEventID, token.LastEventID,
-					serviceerror.NewInternal(fmt.Sprintf("corrupted history event batch, eventID is not continuous"))
+					serviceerror.NewDataLoss(fmt.Sprintf("corrupted history event batch, eventID is not contiguous"))
```

Comment:
Remove `fmt.Sprintf` or add eventIds to the message.

Reply:
the fmt sprintf should be changed during the grpc migration